src/modules/app/containers/Switcher.js

```diff
@@ -21,7 +21,7 @@ import GeneralFooter from '../../layout/components/GeneralFooter'
 import type { StateType } from '../StateType'
 import { getRouteConfig } from '../route-configs'
 import Helmet from '../../common/containers/Helmet'
-import type { Location, Dispatch } from 'redux-first-router'
+import type { Dispatch, LocationState } from 'redux-first-router/dist/flow-types'
```

Comment:
this should be split into
```suggestion
import type { Dispatch } from 'redux'
import type { LocationState } from 'redux-first-router'
```